content/en/internal/configuration-database/users-roles/_index.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "Users and roles"
+weight: 30
+description: >-
+     Guide about user accounts and the roles.
+---
+
+## Introduction
+
+In this section there is an explanation over the user accounts on GOCDB, how to
+manage them, and the roles defined in GOCDB.
```

Comment:
This section seems quite empty...

Reply:
the real content is in the sub-pages; we can use a different wording.

Reply:
ah, then this is fine :)